tools/journal_proto_processor.cpp

```diff
@@ -481,10 +488,17 @@ void JournalProtoProcessor::ProcessRequiredField(
     }
     field_cxx_type_[descriptor] += "*";
```

Comment:
We should have two wrapper lambdas `field_cxx_mode_fn_` and `field_cs_mode_fn_`.